apps/list.c

```diff
@@ -337,6 +337,8 @@ static void list_options_for_command(const char *command)
             continue;
         BIO_printf(bio_out, "%s %c\n", o->name, o->valtype);
     }
+    /* Always output the -- marker since it is sometimes documented. */
+    BIO_printf(bio_out, "- -\n");
```

Comment:
Why the space between the dashes?

Reply:
The first dash is the option, the second is what it takes (nothing).  I agree it is not obvious.

Reply:
Oh, ah!